cmd/compute-domain-kubelet-plugin/nvlib.go

```diff
@@ -319,65 +312,21 @@ func (l deviceLib) parseNVCapDeviceInfo(nvcapsFilePath string) (*nvcapDeviceInfo
 	return info, nil
 }
 
-func (l deviceLib) createComputeDomainChannelDevice(channel int) error {
-	// Construct the properties of the device node to create.
-	path := fmt.Sprintf("/dev/nvidia-caps-imex-channels/channel%d", channel)
-	path = filepath.Join(l.devRoot, path)
-	mode := uint32(unix.S_IFCHR | 0666)
```

Comment:
I see that here we had also picked 666, and hence there's no change. If that is even applying in the same way as before.

Reply:
Yes, it was pulled from there -- the `unix.S_IFCHR` is no longer needed as that is implied / supplied by the runtime when creating the device node.